application/views/questionGroupsAdministration/import_view.php

```diff
@@ -37,6 +37,20 @@
                             <li><?php echo gT("Question attributes:") . $aImportResults['question_attributes'] ?></li>
                          </ul>                        
                     </p>
+
+                    <!-- Warnings -->
+                    <?php if (count($aImportResults['importwarnings'])>0): ?>
+                        <h2 class="warning"><?php eT("Warnings");?>:</h2>
+                        <ul  class="list-unstyled">
+                            <?php
+                                foreach ($aImportResults['importwarnings'] as $warning)
+                                { ?>
+                                <li><?php echo $warning; ?></li>
+                                <?php
+                            } ?>
```

Comment:
endforeach; instead? You're using endif; below already. :)

Reply:
Yes, we were just reusing the same style which was on the file :)